src/firefly/js/visualize/ImagePlotCntlr.js

```diff
@@ -390,12 +409,30 @@ export function dispatchChangeExpandedMode(expandedMode) {
 }
 
 
+
+export function dispatchChangeMouseReadoutReadout1( newRadioValue) {
+
+    flux.process({type: CHANGE_MOUSE_READOUT_READOUT1, payload: {newRadioValue }});
+}
+
+
+export function dispatchChangeMouseReadoutReadout2(newRadioValue) {
+    flux.process({ type: CHANGE_MOUSE_READOUT_READOUT2, payload: {newRadioValue} });
+
```

Comment:
Why don't you make these two actions into 1.  Just make a parameter determine which readout it is.